verification/src/solidity/compiler_fetcher.rs

```diff
@@ -104,15 +108,97 @@ impl TryFrom<(json::CompilerInfo, &Url)> for CompilerInfo {
     }
 }
 
+#[derive(Default)]
+struct RefreshableCompilerVersions {
+    versions: Arc<RwLock<CompilerVersions>>,
```

Comment:
we don't have any awaits inside locks, so make this `parking_lot::RwLock`
also may be include full name, do not `use parking_lot::RwLock`